src/styles/blocks/clock__sec.scss

```diff
@@ -0,0 +1,15 @@
+.clock__sec {
```

Comment:
clock__sec selector has a lot of similar css rules with clock__min. So either create a common class or create placeholder/mixin